### PR TITLE
Update MorpheusVM Tutorial

### PR DESCRIPTION
**Pull request by RodrigoVillar**
This PR updates the MorpheusVM tutorial to reflect the latest changes.

In addition, this PR updates the tutorial to use procedural integration tests and the HyperSDK CLI

### CODE REVIEW

**Comment by RodrigoVillar** [comment]
Note: wait for `hypersdk-cli` package to be updated to include the `balance` command. Once the command is part of the CLI, then we can merge to main.

**Comment by RodrigoVillar** [comment]
> Note: wait for `hypersdk-cli` package to be updated to include the `balance` command. Once the command is part of the CLI, then we can merge to main.

Updated CLI version used in doc; you can now use the `balance` command

File: docs/tutorials/morpheusvm/3_testing.md [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+- Creating a context with a deadline of 2 seconds
+  - If the test takes longer than 2 seconds, it will fail
+- Calling `ConfirmTxs` with our TX being passed in
+
+The function `ConfirmTXs` is useful as it checks that our TX was
+sent and that, if finalized, our transaction has the expected outputs. We have
+the following:
+
+```go
+	timeoutCtx, timeoutCtxFnc := context.WithDeadline(context.Background(), time.Now().Add(2*time.Second))
```
Comment (aaronbuchwald) [inline docs/tutorials/morpheusvm/3_testing.md:415]: Can we update to use 30s?

File: docs/tutorials/morpheusvm/3_testing.md [hunk 2]
```diff
@@ -0,0 +1,515 @@
...
+			consts.Name,
+			vm.NewParser(genesis)),
+		keys: keys,
+	}, nil
+}
+```
+
+We now move onto our procedural tests.
+
+## Implementing our Procedural Tests
```
Comment (aaronbuchwald) [inline docs/tutorials/morpheusvm/3_testing.md:312]: Can we rename from procedural (term used to differentiate from the old gross generator style) to just calling them tests?

I think using the term procedural will just confuse people.

**Review by aaronbuchwald — APPROVED** [review]

File: docs/tutorials/morpheusvm/3_testing.md (hunk 2 above)
Comment (RodrigoVillar) [reply docs/tutorials/morpheusvm/3_testing.md:312]: What do you think about the following syntax:

- "tests": refers to all tests we go over
- "workload tests": self explanatory
- "integration tests": the procedural tests themselves

This way, there's a clear difference between the two types of tests we're implementing

Comment (RodrigoVillar) [reply docs/tutorials/morpheusvm/3_testing.md:312]: 👍

**Review by aaronbuchwald — APPROVED** [review]

**Pull request merged**